### DISCUSSION TITLE
RGB 0-255 Values and HEX Color Code Support

### DISCUSSION THREAD

**Discussion by TheWolfBunny64** [bluecaret/carettab#372] [Ideas and Feature Requests]
Hello, BlueCaret! I've been using your CaretTab extension for a while, but ever since the extension got a new makeover last year, I've been kind of hesitant of using it.

Now let's move on what I'm trying to request. While it's cool you have all these pre-set colors, and the ability to make your own colors with the HSL settings, I really couldn't help but notice something rather... how do I put this... important is missing.

Can I request 6-digit HEX code support to be added in, including 0-255 RGB values? Those to me are the two more important tools in order to insert my own colors, and choose them anytime for CaratTab. Can you please do something about it? Thanks!

**Comment by bluecaret** [comment]
Yes I can! I originally went with HSL only as I thought the option to easily adjust the Lightness and Saturation would be useful for some features that never ended up happening, so locking to just that as an option doesn't make sense anymore. Can't guarantee when, but at some point I'll add in more color code options. Thank you.
